packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md

```diff
@@ -0,0 +1,316 @@
+
+The token is returned by the system in [`application(_:didRegisterForRemoteNotificationsWithDeviceToken:)`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622958-application). Use a property to store the token, though don't save the token between app launches as it can change. If there's an error registering for notifications, the system calls [`application(_:didFailToRegisterForRemoteNotificationsWithError:)`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622962-application). In this case, your app can still [load undelivered challenges](#_load_undelivered_challenges).
+
+Add an App Group Capability to your app if it doesn't already have one. You'll use the group identifier in the next step. For more information on adding and configuring an App Group, see [Configuring App Groups](https://developer.apple.com/documentation/xcode/configuring-app-groups/) in Apple developer documentation.
+
+### Initialize the client
+
+You must initialize the SDK client when your app launches as it configures some Push Notification settings. The following function is an example of configuring and initializing the client that is called from [`application(_:didFinishLaunchingWithOptions:)`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622921-application). The call can go before or after the code to register for push notifications that you added earlier.
+
+```swift
+func initOktaDeviceAuthenticator() {
+    do {
+        let applicationConfig = ApplicationConfig(applicationName: "Your-app-name",
+                                                  applicationVersion: "Your-app-version",
+                                                  applicationGroupId: "com.your.group.id")
+
+        #if DEBUG
+            applicationConfig.pushSettings.apsEnvironment = .development
+        #else
+            applicationConfig.pushSettings.apsEnvironment = .production
+        #endif
+
+        // Set the property that holds the current Devices SDK client.
+        self.authenticatorClient = try DeviceAuthenticatorBuilder(applicationConfig: applicationConfig).create()
+    } catch {
+        // Client not initialized, handle the error.
+    }
+}
+```
+
+Use the name of the group you added when you added the App Group Capability earlier. The compiler conditional ensures that Device Authenticator uses the appropriate APNs environment.
+
+### Register the device
```

Comment:
Because "register" is a simpler word that means the same thing. That's why the first sentence defines enroll for the reader. Enroll is domain specific technical jargon that an enterprise "Ticket" developer may never have heard.